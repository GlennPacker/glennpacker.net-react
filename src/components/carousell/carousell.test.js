import React from 'react';
import '../SetupTests';
import Carousell from './index';
import { shallow } from 'enzyme';


describe('BackEndDevelopment', () => {
    it('renders correctly', () => {
        const imgs = [
            'a.png'
        ];

        const wrapper = shallow(
            <Carousell
                path="path"
                imgs={imgs}
            />
        )

        expect(wrapper).toMatchSnapshot();
    })
})