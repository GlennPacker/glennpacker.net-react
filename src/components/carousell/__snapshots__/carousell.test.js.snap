// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackEndDevelopment renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="pt-3 d-none d-sm-block d-md-none"
  >
    <Slider
      accessibility={false}
      adaptiveHeight={false}
      arrows={false}
      autoplay={true}
      autoplaySpeed={4000}
      dots={false}
      infinite={true}
      slidesToScroll={1}
      slidesToShow={1}
      speed={1000}
    >
      <div
        key="a.png"
      >
        <img
          alt="path certificate a"
          height="250px"
          src="/path/a.png"
        />
      </div>
    </Slider>
  </div>
  <div
    className="pt-3 d-none d-md-block d-xl-none"
  >
    <Slider
      accessibility={false}
      adaptiveHeight={false}
      arrows={false}
      autoplay={true}
      autoplaySpeed={4000}
      dots={false}
      infinite={true}
      slidesToScroll={1}
      slidesToShow={2}
      speed={1000}
    >
      <div
        key="a.png"
      >
        <img
          alt="path certificate a"
          height="250px"
          src="/path/a.png"
        />
      </div>
    </Slider>
  </div>
  <div
    className="pt-3 d-none d-md-none d-lg-none d-xl-block"
  >
    <Slider
      accessibility={false}
      adaptiveHeight={false}
      arrows={false}
      autoplay={true}
      autoplaySpeed={4000}
      dots={false}
      infinite={true}
      slidesToScroll={2}
      slidesToShow={3}
      speed={1000}
    >
      <div
        key="a.png"
      >
        <img
          alt="path certificate a"
          height="250px"
          src="/path/a.png"
        />
      </div>
    </Slider>
  </div>
</div>
`;
